packages/artifact/src/internal/download-http-client.ts

```diff
@@ -201,6 +203,29 @@ export class DownloadHttpClient {
       }
     }
 
+    const isAllBytesReceived = (
+      expected?: string,
+      received?: number
+    ): boolean => {
+      // be lenient, if any input is missing, assume success, i.e. not truncated
+      if (
+        !expected ||
+        !received ||
+        process.env['ACTIONS_ARTIFACT_SKIP_DOWNLOAD_VALIDATION']
```

Comment:
What's the plan with this env variable? Just a fallback if there are issues that users or us can set?

Reply:
Correct, a feature flag sort of thing for now.

Reply:
I'm blind 🤦